.env.dist

```diff
@@ -0,0 +1,25 @@
+###> symfony/framework-bundle ###
```

Comment:
```suggestion
```

not needed and not so true... or if we use it, we have to close it (using `###<` syntax)